test/Microsoft.AspNetCore.Server.Kestrel.Performance/PipeThroughputBenchmark.cs

```diff
@@ -38,7 +38,7 @@ public void ParseLiveAspNetTwoTasks()
 
             var reading = Task.Run(async () =>
             {
-                int remaining = InnerLoopCount * _writeLenght;
+                long remaining = InnerLoopCount * _writeLenght;
```

Comment:
Because of `remaining -= result.Buffer.Length;` it looked better changing this `int` to `long` vs casting,